File: 백준/Bronze/5622. 다이얼/다이얼.py
alphabet = ["ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"]

input_ = input()

ans = 0
for text in input_:
    for idx, tmp in enumerate(alphabet):
       if text in tmp:
           ans += idx + 3
       else:
           pass

print(ans)

File: 백준/Bronze/5622. 다이얼/다이얼.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        char[] text = scanner.nextLine().toCharArray();

        String[] alphabet = {"ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"};

        int ans = 0;
        for (char txt: text)
        {
            for (int idx=0; idx < alphabet.length; idx++)
            {
                for (char tmp: alphabet[idx].toCharArray())
                {
                    if (txt == tmp)
                    {
                        ans += idx + 3;
                    }
                    else
                    {
                        continue;
                    }
                }
            }
        }
        System.out.println(ans);
    }
}